Tests/Remove-CEventLog.Tests.ps1

```diff
@@ -0,0 +1,52 @@
+
+#Requires -RunAsAdministrator
+#Requires -Version 5.1
+Set-StrictMode -Version 'Latest'
+
+BeforeAll {
+    Set-StrictMode -Version 'Latest'
+
+    & (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-Test.ps1' -Resolve)
+
+    $script:logName = 'New-CEventLog.Test'
+
+    Uninstall-CEventLog -LogName $script:logName
+
+    function GivenEventLog
+    {
+        Install-CEventLog -LogName $script:logName -Source 'Carbon.Windows'
+    }
+
+    function WhenRemovingEventLog
+    {
+        Remove-CEventLog -LogName $script:logName -ErrorAction 'SilentlyContinue'
+    }
+
+    function ThenError
+    {
+        $Global:Error[0].Exception.Message | Should -Match 'does not exist.'
+    }
+
+    function ThenEventLogRemoved
+    {
+        [Diagnostics.EventLog]::GetEventLogs().LogDisplayName | Should -Not -Contain $script:logName
+    }
+}
+
+Describe 'Remove-CEventLog' {
+    BeforeEach {
+        $Global:Error.Clear()
+    }
+
+    It 'should remove event log' {
+        GivenEventLog

```

Comment:
Instead of hard-coding a global event log name, make it a required parameter of the `Given`, `When`, and `Then` functions. When you read a test, you shouldn't have to look anywhere else to see the important test data being used.